### ISSUE TITLE
datatable: mat-select for items per page is not accessible when focusing on items

### ISSUE THREAD

**Issue by danw5050** [angular/material#11211]
### Bug, feature request, or proposal:
The mat-select which is used within the datatable to select the number of items in a page is not read by screen readers (ChromeVox and Mac Reader) as the user changes the number which is focus on the extended dropdown.

### What is the expected behavior?
The  screenreader to be able to pick up on the current number (representing items/page within datatable) that is in focus.

### What is the current behavior?
The screenreader does not pick up on the number (representing items/page within datatable) that is in focus.

### What is the use-case or motivation for changing an existing behavior?
To add better accessibility for the datatable

### Which versions of AngularJS, Material, OS, and browsers are affected?
Chrome, Mac

**Comment by Splaktar** [comment]
Please submit Angular Material questions [here](https://groups.google.com/forum/#!forum/angular-material2) and issues [here](https://github.com/angular/material2/issues). This repo is for AngularJS Material.
